diff --git a/dist/plugin.js b/dist/plugin.js
index 87b022d5e3e65c4d684b93c28973fc02ec19f57a..470a071dd6f4bce6bb0cd373682c3e6c1a7f4321 100644
--- a/dist/plugin.js
+++ b/dist/plugin.js
@@ -2625,9 +2625,7 @@ async function executeReadBatch(batchToExecute) {
   const authCompletedFn = await checkOpCliAuth();
   await spawnAsync("op", `run --no-masking ${lockCliToOpAccount ? `--account ${lockCliToOpAccount} ` : ""}-- env -0`.split(" "), {
     env: {
-      // have to pass through at least path so it can find `op`, but might need other items too?
-      PATH: process.env.PATH,
-      // ...process.env as any,
+      ...process.env,
       ...envMap
     }
   }).then(async (result) => {
